node/src/node/builder.rs

```diff
@@ -231,17 +230,19 @@ where
         }
     }
 
-    /// Set pruning delay.
+    /// Set pruning window.
     ///
-    /// Pruning delay defines how much time the pruner should wait after sampling window in
-    /// order to prune the block.
+    /// Pruning window defines maximum age of a block considered to be kept in store.
```

Comment:
```suggestion
    /// Pruning window defines maximum age of a block for it to be retained in store.
```
og makes it sound like we consider block to be in store solely by virtue of its age, rather than the (I think) intended "this defines how long we keep stuff".